google/cloud/bigquery/dbapi/cursor.py

```diff
@@ -192,11 +198,55 @@ def executemany(self, operation, seq_of_parameters):
         for parameters in seq_of_parameters:
             self.execute(operation, parameters)
 
+    def _bqstorage_fetch(self, bqstorage_client, size):
```

Comment:
We aren't actually using `size`, right? I don't believe that's a supported [read option](https://googleapis.dev/python/bigquerystorage/latest/gapic/v1beta1/types.html#google.cloud.bigquery_storage_v1beta1.types.TableReadOptions).

Any reason we have the parameter here? If so, let's document why it's present despite it not being used.

Reply:
No reason, just passed in the arguments to `_try_fetch()` to this method initially. It actually turns out that even `_try_fetch()` ignores it completely.